source/settings/settings-reference.txt

```diff
@@ -0,0 +1,103 @@
+.. _compass-interface-settings:
+.. _compass-settings-reference:
+
+--------------------------
+Compass Interface Settings
+--------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides an overview on options that you can set on the 
+|compass| interface. For more detailed guidance on how to set a specific option, 
+see the task page for the specific option.
+
+Settings
+--------
+
+You can configure the following settings on the |compass| interface:
```

Comment:
[suggestion] Do we need this intro sentence?

Reply:
Had this here to specify that these are the settings you'll be able to set on the interface, rather than the config file or cli. 

Additionally, style guide recommends including [some introductory text](https://www.mongodb.com/docs/meta/style-guide/style/tables/#introductory-text-for-tables) to briefly explain the contents of the table.